app/v6/lossClaims/delete/def1/Def1_DeleteLossClaimValidator.scala

```diff
@@ -17,20 +17,24 @@
 package v6.lossClaims.delete.def1
 
 import cats.data.Validated
-import cats.implicits.catsSyntaxTuple2Semigroupal
+import cats.implicits.catsSyntaxTuple3Semigroupal
 import shared.controllers.validators.Validator
-import shared.controllers.validators.resolvers.ResolveNino
+import shared.controllers.validators.resolvers.{ResolveNino, ResolveTaxYearMinimum}
 import shared.models.errors.MtdError
+import v6.lossClaims.common.minimumTaxYear
 import v6.lossClaims.common.resolvers.ResolveLossClaimId
 import v6.lossClaims.delete.def1.model.request.Def1_DeleteLossClaimRequestData
 import v6.lossClaims.delete.model.request.DeleteLossClaimRequestData
 
-class Def1_DeleteLossClaimValidator(nino: String, claimId: String) extends Validator[DeleteLossClaimRequestData] {
+class Def1_DeleteLossClaimValidator(nino: String, claimId: String, taxYear: String) extends Validator[DeleteLossClaimRequestData] {
+
+  private val resolveTaxYear: ResolveTaxYearMinimum = ResolveTaxYearMinimum(minimumTaxYear)
```

Comment:
This will result in the wrong error. This has to be specific to claims so please update to the below, thanks

```
  private val resolveTaxYearClaimedFor = ResolveTaxYearMinimum(
    minimumTaxYear = minimumTaxYear,
    formatError = TaxYearClaimedForFormatError
  )
```